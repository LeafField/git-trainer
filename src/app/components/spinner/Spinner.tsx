import React from "react";

const Spinner = () => {
  return (
    <div className="my-16 flex justify-center">
      <div
        className={`h-10 w-10 animate-spin rounded-full border-4 border-gray-500 border-t-gray-200`}
      ></div>
    </div>
  );
};

export default Spinner;
