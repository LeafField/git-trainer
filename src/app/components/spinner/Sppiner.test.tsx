import { render } from "@testing-library/react";
import Spinner from "./Spinner";

describe("Spinnerの単体テスト", () => {
  it("スナップショットテスト", () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
