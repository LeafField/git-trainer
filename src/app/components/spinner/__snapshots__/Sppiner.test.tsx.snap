// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinnerの単体テスト スナップショットテスト 1`] = `
<div>
  <div
    class="my-16 flex justify-center"
  >
    <div
      class="h-10 w-10 animate-spin rounded-full border-4 border-gray-500 border-t-gray-200"
    />
  </div>
</div>
`;
